ready/AzNamingTool/Helpers/ConfigurationHelper.cs

```
@@ -22,16 +27,33 @@ public static SiteConfiguration GetConfigurationData()
             return config;
         }
 
-        public static string GetAppSetting(string key)
+        public static string GetAppSetting(string key, bool decrypt = false)
```

Are you ok to swallow the exception and return an empty string in case of failure loading the app config? What side effect would you experience if this happened?

---

Hi Jose, 
This is a tough one as this code is in a helper class. If the site cannot load the configuration, it would essentially cause every line of code to error, as the configuration data is essential to the site running. I believe the site would navigate to a generic error page but would need to do further testing to confirm. In either case, it would be completely unusable if there was a catastrophic configuration loading failure.